src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java

```diff
@@ -24,27 +26,18 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 import java.io.IOException;
-import java.security.NoSuchAlgorithmException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
-import static org.fundacion.jala.converter.models.AssetSQL.insertAssetData;
-import static org.fundacion.jala.converter.models.AssetSQL.listAsset;
-import static org.fundacion.jala.converter.service.ChecksumService.getFileChecksum;
 import static org.fundacion.jala.converter.service.ExtractMetadata.extractMetadata;
-import static org.fundacion.jala.converter.service.ZipService.zipFile;
-import static org.fundacion.jala.converter.service.ZipService.zipFiles;
-
 
 @RestController
 @RequestMapping("/api")
 public class AudioConverterController {
     @Autowired
     private FileStorageService fileStorageService;
     private static final Logger LOGGER = LogManager.getLogger();
+    private ParameterOutputChecksum parameterOutputChecksum;
 
     /**
-     * Endpoint for audio converter
+     * Calls endpoint for audio converter
      */
     @PostMapping("/convertAudio")
     public String uploadFile(final @RequestParam("file") MultipartFile file,

```

Comment:
You should write the parameters that can enter within the limit of 120 characters.